docs/collections/catalog.md

```diff
@@ -17,18 +17,24 @@ tags:
 Below you can find a list of all available Prefect Collections.
 
 <!-- The code below is a jinja2 template that will be rendered by generate_catalog.py -->
-<div style="display:grid; grid-template-columns: repeat(auto-fit, minmax(235px, 1fr));">
+<div style="display:grid; grid-template-columns: repeat(auto-fit, minmax(144px, 1fr));">
```

Comment:
I recommend setting the min width to 165px to prevent OpenMetadata from wrapping
```suggestion
<div style="display:grid; grid-template-columns: repeat(auto-fit, minmax(165px, 1fr));">
```